packages/svelte/.gitignore

```diff
@@ -0,0 +1,11 @@
+.DS_Store
```

Comment:
Same as above, is this necessary to be separate from the `.gitignore` at the root of the repo?